src/patterns/menu/Menu.root.tsx

```diff
@@ -0,0 +1,51 @@
+import type { FlattenSimpleInterpolation, ThemedStyledProps } from 'styled-components';
+import styled, { css } from 'styled-components';
+
+import Spacing, { applyPadding } from '../../utilities/spacing';
+import type { TTheme } from '../../utilities/theme';
+import { applyShape } from '../../foundations/shape';
+import Text, { applyTextMargin, applyTextStyles } from '../../components/text';
+
+import type { PMenuLabelRoot, PMenuRoot } from './Menu.props';
+
+const MenuLabelRoot = styled(Text)<PMenuLabelRoot>(
+    ({ isMobile }): FlattenSimpleInterpolation => css`
+        ${applyTextStyles({ size: 100, weight: 'bold' })};
+        ${applyTextMargin({ margin: 'none' })};
+        ${applyPadding(Spacing.trbl({ top: 0, right: 200, bottom: 100, left: 200 }))};
+        text-align: ${isMobile ? 'center' : 'left'};
+    `
+);
+
+const MenuGroupLabelRoot = styled(Text)<PMenuLabelRoot>(
+    ({ isMobile }): FlattenSimpleInterpolation => css`
+        ${applyTextStyles({ size: 100, weight: 'regular' })};
+        ${applyTextMargin({ margin: 'none' })};
+        ${applyPadding(Spacing.symmetric({ vertical: 75, horizontal: 200 }))};
+        text-transform: uppercase;
+        text-align: ${isMobile ? 'center' : 'left'};
+    `
+);
+
+const MenuRoot = styled.div<PMenuRoot>(
+    ({
+        width,
```

Comment:
those will be passed down to the DOM ... once we decided if we use transient props we can change these as well ... otherwise we would need to add the `withConfig`/`shouldForwardProp` combination here as well